src/app/logging.ts

```diff
@@ -19,7 +19,7 @@ const logFields = (logEvent: any): any => {
     };
 };
 
-addLayout('json', config => {
+addLayout('json', () => {
```

Comment:
How did this get through?!

Reply:
🤷‍♂ The missing no-undef rule bugged me for ages - I fully expect that rule to always be present; my brain gets freaked out when it's not